### PR TITLE
Update streamcap.fr_FR.ts

### PR DESCRIPTION
**Pull request by AzogBiceps**
Some translation done in French

### CODE REVIEW

**Comment by olielvewen** [comment]
@AzogBiceps 

Thanks a lot.

However, I am a bit annoying with the following strings : 
- destination de rendu
- chemin de rendu
- choisir chemin de rendu

For me, I'll said rather  (in the same order) :
- répertoire de sortie
- chemin de (repertoire de) destination
- choisir le répertoire de destination 

what did you think about that ? If you are agree with, could you make modifications ? THX.

**Comment by olielvewen** [comment]
#6

**Comment by olielvewen** [comment]
#3

File: streamcap/streamcap.fr_FR.ts [hunk 1]
```diff
@@ -49,12 +49,12 @@
     </message>
     <message>
         <location filename="streamcap.py" line="180"/>
-        <source>Output Destination</source>
+        <source>Destination de rendu</source>
```
Comment (Toinane) [inline streamcap/streamcap.fr_FR.ts]:
```suggestion
        <source>Destination de sortie</source>
```

File: streamcap/streamcap.fr_FR.ts [hunk 2]
```diff
@@ -49,12 +49,12 @@
...
         <translation type="unfinished"></translation>
     </message>
     <message>
         <location filename="streamcap.py" line="183"/>
-        <source>Output Path</source>
+        <source>Chemin de rendu</source>
```
Comment (Toinane) [inline streamcap/streamcap.fr_FR.ts]:
```suggestion
        <source>Chemin de sortie</source>
```

File: streamcap/streamcap.fr_FR.ts [hunk 3]
```diff
@@ -84,7 +84,7 @@
     </message>
     <message>
         <location filename="streamcap.py" line="271"/>
-        <source>Choose output path</source>
+        <source>Choisir chemin de rendu</source>
```
Comment (Toinane) [inline streamcap/streamcap.fr_FR.ts]:
```suggestion
        <source>Choisir le chemin de sortie</source>
```

**Comment by AzogBiceps** [comment]
The translations provided by Toinane are better than mine, you can use his corrections!

**Comment by olielvewen** [comment]
Yes. I'm agree with you.

**Comment by olielvewen** [comment]
Done. Thanks a lot.

**Pull request merged**